### PR TITLE
[event based orderbook] [easy] Add info logs to event based orderbook

### PR DESCRIPTION
**Pull request by e00E**
This allows to see how long the inital update from past events takes.

### CODE REVIEW

File: driver/src/orderbook/streamed/updating_orderbook.rs [hunk 1]
```diff
@@ -117,9 +119,12 @@ async fn update_with_events_forever(
                 handle_event(&orderbook, &mut block_timestamp_reader, event).await?;
             },
             past_events = past_events => {
-                for event in past_events? {
+                let past_events = past_events?;
+                log::info!("Got {} past events.", past_events.len());
```
Comment (nlordell) [inline driver/src/orderbook/streamed/updating_orderbook.rs:123]: Can we add an info event for new events as well?

**Review by nlordell — APPROVED** [review]
Easy! Just a small extra logging request.

**Review by fleupold — APPROVED** [review]

File: driver/src/orderbook/streamed/updating_orderbook.rs (hunk 1 above)
Comment (bh2smith) [inline driver/src/orderbook/streamed/updating_orderbook.rs:123]: NIT:
```suggestion
                log::info!("Retrieved {} past events.", past_events.len());
```

**Review by bh2smith — APPROVED** [review]
💙

**Pull request merged**